packages/frontend/sample.env

```diff
@@ -13,4 +13,6 @@ REACT_APP_FF_PRECINCTS=false
 REACT_APP_FF_MULTI_RACE=false
 REACT_APP_FF_MULTI_WINNER=true
 REACT_APP_FF_CUSTOM_REGISTRATION=false
-REACT_APP_FF_VOTER_FLAGGING=false
\ No newline at end of file
+REACT_APP_FF_VOTER_FLAGGING=false
+REACT_APP_FEATURED_ELECTIONS=4e7b3f9d-ce53-4b25-8747-b5927c6745aa,753bb873-12de-4aca-af25-e96ec72f0b49,18ad9f37-94a8-4fed-a784-83761c692052
```

Comment:
Thanks for adding this, this has this has been lacking in the onboarding instructions for a while